README.md

```diff
@@ -1,2 +1,116 @@
-# eth-client
-A Golang Client Library for Communicating with Ethereum RPC Servers
+eth-client
+==========
+
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg)](https://travis-ci.org/getamis/eth-client)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Go Report Card](https://goreportcard.com/badge/github.com/alanchchen/web3go)](https://goreportcard.com/report/github.com/getamis/eth-client)
+
+A Golang Client Library for Communicating with Ethereum RPC Servers. 
```

Comment:
`A Golang client library to communicate with Ethereum RPC server. `

Reply:
OK